packages/patternfly-4/react-core/src/components/Popover/Popover.tsx

```diff
@@ -24,84 +27,84 @@ export const PopoverPosition = {
   right: 'right'
 };
 
-const propTypes = {
+export interface PopoverProps {
   /** Popover position */
-  position: PropTypes.oneOf(Object.keys(PopoverPosition).map(key => PopoverPosition[key])),
+  position?: BasicPlacement;
```

Comment:
Since we import this react-docgen can't show the types for it in the docs. I'd prefer `'top' | 'bottom' | 'left' | 'right'` for now.

Reply:
fixed